global_helpers/test_panther_greynoise_helpers.py

```diff
@@ -0,0 +1,113 @@
+import pytest
+
+from panther_greynoise_helpers import (
+    GreyNoiseAdvanced,
+    GreyNoiseBasic,
+    PantherIncorrectIPAddressMethodException
+)
+import json
+
+test_data_basic_list = [(
+    {
+        "ip_address": "2.2.2.2",
+        "request_user":
+        "test","request_time":
+        "time","p_enrichment": {
+            "greynoise_noise_basic": {
+                "p_any_ip_addresses": [
+                    "8.8.8.8",
+                    "1.1.1.1",
+                    "localhost",
+                    "2.3.4.5",
+                ],
+            }
+        }
+    }
+)]
+
+test_data_basic_str = [(
+    {
+        "ip_address": "2.2.2.2",
+        "request_user": "test",
+        "request_time": "time",
+        "p_enrichment": {
+            "greynoise_noise_basic": {
+                "p_any_ip_addresses": {
+                    "actor": "unknown", "ip": "8.8.8.8","classification": "unknown"
+                },
+            }
+        }
+    }
+)]
+
+test_data_advanced_list = [(
+    {
+        "ip_address": "2.2.2.2",
+        "request_user": "test",
+        "request_time": "time",
+        "p_enrichment": {
+            "greynoise_noise_advanced": {
+                "p_any_ip_addresses": [
+                    {"actor": "unknown", "ip": "8.8.8.8","classification": "unknown"},
+                ],
+            }
```

Comment:
Why `p_any_ip_addresses` mapping to a list of dictionaries when the test basic list has it mapped to a list of strings?

Reply:
No reason in particular, though in both cases they should return a list.